x-pack/plugins/dashboard_enhanced/public/services/drilldowns/actions/flyout_create_drilldown/flyout_create_drilldown.tsx

```diff
@@ -126,6 +126,7 @@ export class FlyoutCreateDrilldownAction implements Action<EmbeddableApiContext>
         core
       ),
       {
+        maxWidth: 500,
```

Comment:
**nit** Could we make this a constant so that `flyout_create_drilldown` and `flyout_edit_drilldown` stay in sync? :eyes:

Reply:
Makes sense to me! Changes in [0200bf9](https://github.com/elastic/kibana/pull/189009/commits/0200bf953065173e63becd0d2464353580a83f50)